.github/workflows/preview-html.yml

```diff
@@ -41,8 +39,36 @@ jobs:
     - uses: Azure/k8s-deploy@v1
       with:
         manifests: |
-          kubernetes/covid19inv-cj.yml
+          kubernetes/covid-19-guidance-deployment.yaml
+          kubernetes/covid-19-guidance-service.yaml
         images: |
-          tbsacr.azurecr.io/covidinventorycreator:${{ github.sha }}
+          tbsacr.azurecr.io/covid-19-guidance:${{ github.sha }}
         namespace: |
-          covid19inv
+          covid-19-guidance
+
+    # Grab a list of changed files to preview (HTML)
+    - uses: lots0logs/gh-action-get-changed-files@2.1.4
+      id: printing
+      with:
+        token: '${{ secrets.GITHUB_TOKEN }}'
+        paths: '*.html'
+        format: space-delimited
+    - name: Printing
+      run: |
+        ./.github/workflows/parse-file-array.sh
+
+    #- name: Test
+    #  run: |
+    #    echo ${{ env.LINKS }}
+    
+    # Create comment in the PR with the urls of the changed files
+    - name: Update status to Waiting for approval
```

Comment:
Aside: you might want to check this out if you want to only have one comment on the PR that gets updated https://github.com/w3c/aria-practices/blob/78bb9a05a39a1fdfe5971d692e149cf65ae93780/.github/workflows/coverage-report.yml#L26